pkg/asset/installconfig/aws/regions.go

```diff
@@ -14,12 +14,10 @@ func knownRegions(architecture types.Architecture) map[string]string {
 	required := rhcos.AMIRegions(architecture)
 
 	regions := make(map[string]string)
-	for _, partition := range endpoints.DefaultPartitions() {
-		for _, partitionRegion := range partition.Regions() {
-			partitionRegion := partitionRegion
-			if required.Has(partitionRegion.ID()) {
-				regions[partitionRegion.ID()] = partitionRegion.Description()
-			}
+	for _, partitionRegion := range endpoints.AwsPartition().Regions() {
+		partitionRegion := partitionRegion
+		if required.Has(partitionRegion.ID()) {
+			regions[partitionRegion.ID()] = partitionRegion.Description()
```

Comment:
nit: Change `partitionRegion` to `region`. The partition part of the name is not informative.
Also, I have no idea what the intention of the following line was.
```
		partitionRegion := partitionRegion
```
```suggestion
	for _, region := range endpoints.AwsPartition().Regions() {
		if required.Has(region.ID()) {
			regions[region.ID()] = region.Description()
```